### ISSUE TITLE
Bug with relations for siblings in online project not updated

### ISSUE THREAD

**Issue by tobias-karrer** [alkacon/opencms-core#805]
Setup: There are two container pages that are siblings of each other: sibling1 and sibling2 - both pages contain contents and are published.

After adding a new content resource on sibling1 and publishing the page, sibling2 will be published automatically as well.
But reading relations for the content resource in online project doesn't return relation to sibling2.

Even after changing the same content resource on sibling2 and publishing the page, the relations for the content resource in online project doesn't return relation to sibling2:
![image](https://github.com/user-attachments/assets/b15ff799-f3a5-404f-9e83-ef8d9eabce4a)
![image](https://github.com/user-attachments/assets/0722f9d9-5ee5-4987-9cca-1d03f1dd88a3)

Problem is database table CMS_ONLINE_RESOURCE_RELATIONS is not updated, even though sibling2 was published automatically (as it is sibling of sibling1)

Only after editing and publishing sibling2 explicitely both siblings are list on next content resource change publishing.
![image](https://github.com/user-attachments/assets/0ca8f6a1-39b2-4896-8aaa-290beed06465)

Looks like it is related to this bug here: https://github.com/alkacon/opencms-core/issues/797

**Comment by tobias-karrer** [comment]
As far as I can see the bug still exists - see new ticket https://github.com/alkacon/opencms-core/issues/806
